Makefile

```diff
@@ -111,6 +111,15 @@ libtwin.a_files-y += backend/linux_input.c
 endif
 
 # Standalone application
+ifeq ($(CONFIG_PERFORMANCE_TEST), y)
```

Comment:
Rewrite as following:
```
target-$(CONFIG_PERFORMANCE_TEST) += perf
```

The performance test program has nothing with "demo."